src/components/pages/register/form/index.tsx

```diff
@@ -79,11 +80,14 @@ function SignUpForm() {
 
   const handleOnSubmit = async (v) => {
     const payload = {
-      ...v,
-      verificationType:
-        v.email && v.mobileNumber ? v.verificationType : VERIFICATION_TYPE[v.email ? 0 : 1].value
+      credentials: {
+        ...v,
+        verificationType:
+          v.email && v.mobileNumber ? v.verificationType : VERIFICATION_TYPE[v.email ? 0 : 1].value
+      },
+      groupId: id
```

Comment:
This is not part of push notification feature can you seprate PR and remove it from here? and clean up this logic? something like 
```
const {email, mobileNumber, verificationType} = v;
const verificationType = email && mobileNumber ? verificationType : VERIFICATION_TYPE[email ? 0 : 1]?.value
```